### DISCUSSION TITLE
Version 2.991

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1310] [General]
## What's Changed
* Optimize internal database, make it faster and smaller.
* Make subscription feed appear (nearly) instantly and do the slow network fetches in the background.
* Use Material themed subscribe button.
* Search query text is clickable on the result.
* Fix broken compilation due to JCenter disapearance
* Show deleted/blocked channels differently, and allow to unsubscribe from them.
* Updated translations for:
  * Catalan, Malayalam, Tamil
* Translators:
  * ek387587, Rohan Kishore, Onebyone

**Full Changelog**: https://github.com/SkyTubeTeam/SkyTube/compare/v2.990...v2.991

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/v2.991'>Version 2.991</a>.</em>

**Comment by hayabuza48** [comment]
How to fix IOS played not supported in android 4.4 after using this app?
